index.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/2e8f9696de.js" crossorigin="anonymous"></script>    <link rel="stylesheet" href="style.css">
+    <title>Home Page</title>
+</head>
+<body>
+  <div class="grid-container">
+    <nav class="navbar">
+        <ul class="social-list large-screen">
+          <li class='social-item'><img src="images/facebook.png" alt="facebook logo"></li>
+          <li class='social-item'><img src="images/twitter.png" alt="twitter logo"></li>
+          <li class='social-item'><a href="#">English</a></li>
+          <li class='social-item'><a href="#">My Page</a></li>
+          <li class='social-item'><a href="#">Logout</a></li>
+        </ul>   
+      <div class="local-nav">
+        <div class="hamburger small-screen">
+          <span class="hamburger-bar"></span>
+          <span class="hamburger-bar"></span>
+          <span class="hamburger-bar"></span>
+        </div>  
+        <a href="#" class="brand"><img src="images/logo.png" alt="logo"  class="brand">Nyegenyege</a>
+        <ul class="nav-list flex-row">
+          <li class="nav-item "><a href="#" class="active">About</a></li>
+          <li class="nav-item"><a href="#">program</a></li>
+          <li class="nav-item"><a href="#">Join</a></li>
+          <li class="nav-item"><a href="#">Sponsor</a></li>
+          <li class="nav-item"><a href="#">News</a></li>
+          <li class="nav-item " ><a href="#">Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+    <div class="home">
+      <header class="title">
+        <p class="tagline spacing">"MTN Nyegenyege welcomes all"</p>
+        <h1 class="page-title page-title-1 spacing">THE MTN NYEGENYEGE FESTIVAL 2022</h1>
+        <p class="introduction spacing">
+          Nyege Nyege is a collective in Kampala, Uganda that 
+          promotes outsider music, primarily electronic, by African artists. 
+          It was founded in 2013 by expats Arlen Dilsizian and Derek Debru. 
+          In addition to organizing ongoing parties and an artist residency
+          the collective runs.
+        </p>
+        <div class="event-date spacing">2022.8.12 (MON) ~ 18(SUN)</div>
+        <p class="venue spacing">The event will be held in Jinja at the Nile</p>
+      </header>
+      <section class="main-program">
+        <h2 class="program-title">main program</h2>
```

Comment:
[ OPTIONAL ]: It would be better if you add some margin up for this section so that the header can take the whole page ✔😀